client/src/components/Notification.js

```diff
@@ -12,6 +13,14 @@ const readableTimeAgo = (time) => {
   return daysAgo > 0 ? `${daysAgo} days ago` : 'today'
 }
 
+const notificationViewerIsResourceRequester = (notification, user) => {
```

Comment:
you can also use isResourceRequester from useUser and pass that function the material_request

Reply:
Smart. Done